app/components/works/buttons_component.rb

```diff
@@ -33,5 +33,10 @@ def model
     def work_in_reviewed_coll?
       collection.review_enabled?
     end
+
+    sig { returns(T::Boolean) }
+    def version_draft?
+      work_version.version_draft? && work_version.persisted?
```

Comment:
Can you explain why this is necessary?

Reply:
Since it hasn't been saved (because an error was returned on form submission), the version draft cannot be discarded. The cancel button is still available for cancelling.

Reply:
Would you be able to rename this as `display_cancel_button?` I think that would make it more clear why we care about the persistence.